### PR TITLE
Feature/#134249293/redesign gaming screen ui

### PR DESCRIPTION
**Pull request by andela-oakinseye**
#### What does this PR do?
Redesign the gaming UI

#### Description of Task to be completed?
Redesign the gaming user interface as well as fix minor issue with the authentication forms.

#### How should this be manually tested?
Run the app using `npm start`  then start a new game to see the redesigned interface.

#### Any background context you want to provide?
N / A

#### What are the relevant pivotal tracker stories?
#134249293 redesign the actual gaming ui.

#### Screenshots (if appropriate)
screen shot 2016-12-14 at 11 10 59 am
screen shot 2016-12-14 at 11 11 34 am

#### Questions:
N/A

### CODE REVIEW

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -8,7 +8,7 @@ script(type='text/javascript', src='/lib/jquery/jquery.js')
 script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 
 //Bootstrap
-script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
+//script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
```
Comment (andela-aolaniran) [inline app/views/includes/foot.jade]: we can remove commented code if we aren't using it.

File: public/css/forms.css [hunk 2]
```diff
@@ -1,10 +1,8 @@
-body {
-  padding: 20px;
-}
 .signup-wrapper {
   border-radius: 2px;
   margin: 0 auto;
   float: none;
+
```
Comment (andela-aolaniran) [inline public/css/forms.css]: Why the space ?

File: app/views/includes/foot.jade (hunk 1 above)
Comment (andela-oakinseye) [reply app/views/includes/foot.jade]: Thanks @andela-aolaniran.
I will fix this now.

**Comment by andela-aolaniran** [comment]
Looks good.

**Comment by raphaelakpan** [comment]
Good job Wale. However, the way you displayed the 2 screenshots appear as if they are one. Try separating them visibly so that it is easy to understand.

**Comment by fuzzybrows** [comment]
Good Job Wale. I think a screenshot showing the answer cards should be included.

**Comment by andela-oakinseye** [comment]
Thanks for the feedback Oreoluwa.

**Pull request closed without merging**